### DISCUSSION TITLE
Change page title

### DISCUSSION THREAD

**Discussion by nikoo-eshratabadi** [laravel/nova-issues#6433] [Ideas & Feature Requests]
Laravel Version: 10.10
Nova Version: 4.25.1
PHP Version: 8.1.23

Description:
I am trying to change the page title for each resource. I know i can override the label() function but i would like to hide the resource label on the index page and still have the page title to show something for example the singular label.

**Comment by vesper8** [comment]
Did you figure this out? Trying to do the same thing. I want to change the text show in the browser tab specifically. Thanks!

**Reply by nikoo-eshratabadi** [reply]
i ended up doing something really messy but try this function in the nova resource 
    
`public static function label()
    {
    }`
